src/server/ApiRoutes/Search.js

```diff
@@ -29,9 +29,9 @@ const createAPIQuery = basicQuery({
 
 const nyplApiClientCall = (query, urlEnabledFeatures = []) => {
   const requestOptions =
```

Comment:
This is exactly the same as on [Bib.js](https://github.com/NYPL/discovery-front-end/pull/1911/files#diff-d6d9946fcb66a10d0a71c534dcb8caeeaa342a22c3e7788e16163e5af0c19076L16). Could we get this header from a shared location?

Reply:
[This code](https://github.com/NYPL/discovery-front-end/pull/1911/files#diff-d6d9946fcb66a10d0a71c534dcb8caeeaa342a22c3e7788e16163e5af0c19076L16-L20) in the Bib.js file is the exact same (less the " || []" on the urlEnabledFeatures) as [this code](https://github.com/NYPL/discovery-front-end/pull/1911/files#diff-9b0f0223002def934948256e36faa2b685bff2ed5555f7a31ef5bc5576c55774R31-R35) in the Search.js file.

To keep things simple and to avoid making the same change in multiple places in the future it would be awesome if the above code was in a shared location rather than duplicated in both files.